azerothcore/azerothcore-wotlk
# fix(DB/*_loot_template): Fill in comment fields

## Changes Proposed:
Fill in `comment` fields for all `*_loot_template` tables.

## Tests Performed:
<details><summary>Before (skinning_loot_template)</summary>

![image](https://user-images.githubusercontent.com/61268368/103162717-4b772f00-47ca-11eb-8485-50c0ab355a9e.png)
</details>

<details><summary>After (skinning_loot_template)</summary>

![image](https://user-images.githubusercontent.com/61268368/103162723-516d1000-47ca-11eb-9834-dc0d8c46a016.png)
</details>

## How to Test the Changes:
Run the SQL
Check comment fields in all *_loot_template tables.

creature_loot_template
disenchant_loot_template
fishing_loot_template
gameobject_loot_template
item_loot_template
milling_loot_template
pickpocketing_loot_template
prospecting_loot_template
reference_loot_template
skinning_loot_template
spell_loot_template

## Target Branch(es):


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
